hermes/probe/probe.go

```diff
@@ -33,71 +33,64 @@ import (
 	cpmetrics "github.com/google/cloudprober/metrics"
 	"github.com/google/cloudprober/probes/options"
 	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+	"github.com/googleinterns/step224-2020/hermes/probe/target"
 
 	probepb "github.com/googleinterns/step224-2020/config/proto"
 	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
 )
 
-// Target holds all of the required information and state for a given target run.
-type Target struct {
-	// Target stores the proto config for the target to be probed.
-	Target *probepb.Target
-
-	// Journal stores the state of MonitorProbe as a combination of a next operation intent enum and a filenames map.
-	Journal *journalpb.StateJournal
-
-	// LatencyMetrics stores the api call and probe operation latency for a given target run.
-	// Metrics are stored with additional labels to record operation type and exit status.
-	LatencyMetrics *metrics.Metrics
-}
+const (
+	// TODO(evanSpendlove): Refactor to use constant from metrics.go
+	probeLatency = "hermes_probe_latency_seconds"
+)
 
 // Probe holds aggregate information about all probe runs, per-target.
 // It also holds the config and options used to initialise the probe.
 type Probe struct {
 	name    string
 	config  *probepb.HermesProbeDef
-	targets []*Target
+	targets []*target.Target
 	opts    *options.Options
 	logger  *logger.Logger
 }
 
 // interval returns the probing interval in seconds as a time.Duration.
 // Returns:
 //	- time.Duration: returns the probing interval
-func (mp *Probe) interval() time.Duration {
-	return time.Duration(mp.config.GetIntervalSec()) * time.Second
+func (p *Probe) interval() time.Duration {
+	return time.Duration(p.config.GetIntervalSec()) * time.Second
 }
 
 // timeout returns the probe timeout in seconds as a time.Duration.
```

Comment:
"in seconds" contradicts the "as a time.Duration". 
Same on line 57.

Reply:
Okay, I've changed this accordingly.